package.json

```diff
@@ -86,6 +86,7 @@
     "cli-progress": "^3.12.0",
     "codemirror": "^6.0.1",
     "compression": "^1.7.4",
+    "compute-baseline": "^0.1.1",
```

Comment:
Comment: Why don't they publish these packages under a namespace?

Reply:
Not sure, will ask upstream